jenkins/agent-ctl.sh

```diff
@@ -0,0 +1,248 @@
+#!/bin/bash
+# Copyright 2016 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Run command with no args for usage instructions.
+
+usage() {
+  echo "Usage: $(basename "${0}") [--large] <instance> [AUTO|create|delete|update|reboot|attach|detatch]"
```

Comment:
and is there a pattern for instance name? or will it automatically choose one for me?

when would I want to use `--large`?